src/msg/async/AsyncConnection.cc

```diff
@@ -2064,15 +2064,7 @@ void AsyncConnection::fault()
   replacing = false;
   is_reset_from_peer = false;
   outcoming_bl.clear();
```

Comment:
it's expected. the goal is to solve stateful policy connection lose message problem